src/app/core/services/github.service.ts

```diff
@@ -39,6 +39,11 @@ let octokit = new Octokit();
 @Injectable({
   providedIn: 'root',
 })
+
+/**
+ * Responsible for communicating with GitHub to create, update, read and delete
```

Comment:
I think we can generalize it to Github services in general and not just issues, since there are other functionalities involved e.g. repo/label creation

Reply:
yea this would be better, thank you for the suggestion! :D